ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/priv/PrivGetLogs.java

```diff
@@ -85,9 +100,17 @@ public JsonRpcResponse response(final JsonRpcRequestContext requestContext) {
         requestContext.getRequest().getId(), new LogsResult(matchingLogs));
   }
 
+  private List<LogWithMetadata> getEmptyList() {
+    return Collections.emptyList();
+  }
+
   private void checkIfPrivacyGroupMatchesAuthenticatedEnclaveKey(
```

Comment:
We could name this method something like `checkIfAuthenticatedUserWasMemberAtBlock`. This way we reduce the cognitive gap. Not a big deal though.

Or if you like `checkIfPrivacyGroupMatchesAuthenticatedEnclaveKey ` keep it!

Reply:
memberAtBlock makes more sense.